property.go

```diff
@@ -0,0 +1,89 @@
+package opslevel
+
+import "fmt"
+
+type PropertyDefinitionInput struct {
+	Name   string     `json:"name"`
+	Schema JSONString `json:"schema"`
+}
+
+type PropertyDefinition struct {
+	Aliases []string `graphql:"aliases" json:"aliases"`
+	Id      ID       `graphql:"id" json:"id"`
+	Name    string   `graphql:"name" json:"name"`
+	Schema  JSON     `json:"schema" scalar:"true"` // Do not add graphql struct tag here
```

Comment:
well you can add `graphql` struct tags if you neeed - you actually just need `scalar:"true"` because `JSON is a custom scalar type we wrote.